docs/hydra/sdk/01_index.md

```diff
@@ -11,30 +11,36 @@ make backwards compatibility promises with regards to the generated SDKs. We hop
 Before you check out the SDKs, head over to the [REST API](reference/api.mdx) documentation which includes code samples for common
 programming languages for each REST endpoint.
 
+:::info
+
+`ory-hydra-client` SDK has been deprecated, please use `ory-client`!
```

Comment:
Do we have anything on `ory-client`? Are these SDKs officially deprecated?

Reply:
The `ory-client` sdks still exist but they are only for the versions <= `0.2.0` since nearly nobody uses this version anymore its imo more apropriate to link the most recent SDK. But no they are not deprecated if you use Hydra v0.2.0 they are perfectly valid

Reply:
I think the notice is more confusing then helpful, I removed it. 
We should just make a proper announcement that you should move to the ory-client if possible.